recommendation/views.py

```diff
@@ -0,0 +1,61 @@
+from pprint import pprint
+
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.shortcuts import render, get_object_or_404
+from django.urls import reverse_lazy
+from django.views.generic import ListView, CreateView, DeleteView, UpdateView, DetailView
+
+from .helpers import get_related_entries, get_entry_tags
+from .forms import EntryForm, TagForm
+from .models import Entry, Tag
+
+
+class ListEntriesView(ListView):
+    model = Entry
+
+    def get_queryset(self):
+        return Entry.objects.all().order_by('-creation_date')  # minus-sign => descending order
```

Comment:
Possible to define order in model and avoid whole view by passing model as argument to ListView in urls?